tests/forte/data/audio_annotation_test.py

```diff
@@ -143,39 +153,65 @@ def test_audio_annotation(self):
-                
-                self.assertTrue('Recording' in raw_data.keys() and
-                                "recording_class" in raw_data['Recording'])
-                self.assertTrue('AudioUtterance' in raw_data.keys() and
-                                    "speaker" in raw_data['AudioUtterance'])
+
+                self.assertTrue(
+                    "Recording" in raw_data.keys()
+                    and "recording_class" in raw_data["Recording"]
+                )
+                self.assertTrue(
+                    "AudioUtterance" in raw_data.keys()
+                    and "speaker" in raw_data["AudioUtterance"]
+                )
                 # test grouped data
                 if isinstance(data_instance, Recording):
-                    self.assertTrue(array_equal(np.array([data_instance.audio]), raw_data['Recording']['audio']))
-                    self.assertTrue(data_instance.recording_class ==np.squeeze(raw_data['Recording']['recording_class']).tolist())
+                    self.assertTrue(
+                        array_equal(
+                            np.array([data_instance.audio]),
+                            raw_data["Recording"]["audio"],
+                        )
+                    )
+                    self.assertTrue(
+                        data_instance.recording_class
+                        == np.squeeze(
+                            raw_data["Recording"]["recording_class"]
+                        ).tolist()
+                    )
                 elif isinstance(data_instance, AudioUtterance):
-                    self.assertTrue(array_equal(np.array([data_instance.audio]), raw_data['AudioUtterance']['audio']))
-                    self.assertTrue(data_instance.speaker
-                                ==raw_data['AudioUtterance']['speaker'][0])
+                    self.assertTrue(
+                        array_equal(
+                            np.array([data_instance.audio]),
+                            raw_data["AudioUtterance"]["audio"],
+                        )
+                    )
+                    self.assertTrue(
+                        data_instance.speaker
+                        == raw_data["AudioUtterance"]["speaker"][0]
+                    )
 
             # check non-existence of non-requested data fields
             raw_data_generator = pack.get_data(AudioAnnotation)
             for raw_data in raw_data_generator:
                 self.assertFalse("Recording" in raw_data)
                 self.assertFalse("AudioUtterance" in raw_data)
-            
+
             # Check Recording
             recordings = list(pack.get(Recording))
             self.assertEqual(len(recordings), 1)
-            self.assertTrue(array_equal(recordings[0].audio, pack.audio))
-
+            self.assertTrue(
+                array_equal(
+                    recordings[0].audio,
+                    pack.get_payload_data_at(Modality.Audio, 0),
```

Comment:
The same here. If the audio_payload_index is `0`, we can just use `pack.audio`

Reply:
`pack.audio` is removed in this PR.